src/pages/EnablePayments/TermsStep.js

```diff
@@ -78,48 +70,42 @@ class TermsStep extends React.Component {
                             Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                         </Text>
                     </ScrollView>
-                    <View style={[styles.flexRow, styles.mb4]}>
-                        <Checkbox
-                            isChecked={this.state.hasAcceptedDisclosure}
-                            onClick={this.toggleDisclosure}
-                        />
-                        <TouchableOpacity
-                            onPress={this.toggleDisclosure}
-                            style={toggleStyles}
-                        >
+                    <CheckboxWithLabel
+                        style={styles.mb4}
+                        isChecked={this.state.hasAcceptedDisclosure}
+                        onPress={this.toggleDisclosure}
+                        LabelComponent={() => (
                             <Text>
                                 {`${this.props.translate('termsStep.haveReadAndAgree')} `}
 
                                 <TextLink href="https://use.expensify.com/fees">
```

Comment:
Not sure if it's safe to put a `Pressable` inside a `Text`, but I don't think you're supposed to.

Reply:
I think this can (and should) be un-nested

Reply:
Ok I'm not entirely sure either. But this flow isn't being used yet so if it's broken because of this. We'll find out eventually 😂